recipes/relion/build.sh

```diff
@@ -0,0 +1,7 @@
+CUSTOM_CMAKE_ARGS="-DFORCE_OWN_FLTK=OFF"
+CUSTOM_CMAKE_ARGS+=" -DFORCE_OWN_FFTW=OFF"
+CUSTOM_CMAKE_ARGS+=" -DBUILD_SHARED_LIBS=OFF"
```

Comment:
```suggestion
CUSTOM_CMAKE_ARGS+=" -DBUILD_SHARED_LIBS=ON"
```

Any reason we want static libraries? conda-forge strongly prefers shared libraries.

Reply:
@jaimergp as far as I understand this flag controls whether or not RELION builds its own shared libraries, not whether external libraries are statically linked - I have been unable to build RELION with this flag set to OFF 😞

Reply:
True, but that defies the purpose of shipping shared libraries to begin with 😬 And you'd need to find every license of every project ever linked into the static library. It's more fun to debug the shared library issue, trust me hah

Let's see what happens if we switch that off, I'll check the logs and see what might be wrong or who can help!